src/ConseiljsRe.re

```diff
@@ -0,0 +1,185 @@
+open Type;
```

Comment:
Does this belong in ConseilJS instead?

Reply:
I made the types based on `conseiljs`.
But, those are not correct, because we used `any` type a lot in `conseiljs`.
The types are also not correct in `coseiljs`.
So, it is very hard to check types correctly, now.
So I am only following the `mininax`, now.